### PR TITLE
Add LICENSE, CODE_OF_CONDUCT, tweak README

### PR DESCRIPTION
**Pull request by BigBlueHat**
Hi @WileyLabs/teasy !

I've added the necessary files to officially mint this as an open source project.

Please let me know if you have any questions and merge it at your leisure!

Cheers!
🎩

### CODE REVIEW

**Comment by vefimofff** [comment]
@BigBlueHat thanks for this pull request, all in all it looks great. One questions about the license though. After some discussions with Fedor I expected it to be an MIT and I stated so in the pom.xml file. Do you think we need a quick chat with 3 of us to agree on a license?
If it will be Apache, then I would kindly ask to modify the following part in the pom.xml accordingly:

1
--
<licenses>
  | <license>
  | <name>MIT License</name>
  | <url>http://www.opensource.org/licenses/mit-license.php</url>
  | </license>
  | </licenses>

**Comment by BigBlueHat** [comment]
@vefimofff my bad! I copied in the wrong file from another WileyLabs project. 😝 It should be fixed in the latest variation of that commit on this PR.

Thanks for verifying the work earlier! 😁

File: README.md [hunk 1]
```diff
@@ -3,10 +3,18 @@ Test easy with Teasy - UI automation testing framework
...
 </dependency>
 ```
+
+# Contributing
+
+We're just getting started here, but please post issues if you have any!
+
+# License
+
+Apache License 2.0
```
Comment (vefimofff) [inline README.md]: and just a tiny fix here to state MIT to make it 👍

**Comment by vefimofff** [comment]
@BigBlueHat  nevermind, will change it myself as it's a 1 minute thing and I would like to add some more to the readme ;)

**Comment by BigBlueHat** [comment]
Sorry I forgot the README! Thanks @vefimofff

**Pull request merged**